Kitodo/src/main/java/org/kitodo/production/forms/createprocess/ProcessDataTab.java

```diff
@@ -0,0 +1,278 @@
+     */
+    public List<String> getDigitalCollections() {
+        return digitalCollections;
+    }
+
+    /**
+     * Set digitalCollections.
+     *
+     * @param digitalCollections as java.util.List of String
+     */
+    public void setDigitalCollections(List<String> digitalCollections) {
+        this.digitalCollections = digitalCollections;
+    }
+
+    /**
+     * Get availableDigitalCollections.
+     *
+     * @return value of availableDigitalCollections
+     */
+    public List<String> getAvailableDigitalCollections() {
+        return availableDigitalCollections;
+    }
+
+    /**
+     * Set availableDigitalCollections.
+     *
+     * @param availableDigitalCollections as java.util.List of String
+     */
+    public void setAvailableDigitalCollections(List<String> availableDigitalCollections) {
+        this.availableDigitalCollections = availableDigitalCollections;
+    }
+
+    /**
+     * Get tifHeaderImageDescription.
+     *
+     * @return value of tifHeaderImageDescription
+     */
+    public String getTifHeaderImageDescription() {
+        return tifHeaderImageDescription;
+    }
+
+    /**
+     * Set tifHeaderImageDescription.
+     *
+     * @param tifHeaderImageDescription as java.lang.String
+     */
+    public void setTifHeaderImageDescription(String tifHeaderImageDescription) {
+        this.tifHeaderImageDescription = tifHeaderImageDescription;
+    }
+
+    /**
+     * Get tifHeaderDocumentName.
+     *
+     * @return value of tifHeaderDocumentName
+     */
+    public String getTifHeaderDocumentName() {
+        return tifHeaderDocumentName;
+    }
+
+    /**
+     * Set tifHeaderDocumentName.
+     *
+     * @param tifHeaderDocumentName as java.lang.String
+     */
+    public void setTifHeaderDocumentName(String tifHeaderDocumentName) {
+        this.tifHeaderDocumentName = tifHeaderDocumentName;
+    }
+
+    /**
+     * Get guessedImages.
+     *
+     * @return value of guessedImages
+     */
+    public int getGuessedImages() {
+        return guessedImages;
+    }
+
+    /**
+     * Set guessedImages.
+     *
+     * @param guessedImages as int
+     */
+    public void setGuessedImages(int guessedImages) {
+        this.guessedImages = guessedImages;
+    }
+
+    /**
+     * reset all process data.
+     */
+    public void resetProcessData() {
+        this.standardFields = new HashMap<>();
+        this.standardFields.put("collections", true);
+        this.standardFields.put("doctype", true);
+        this.standardFields.put("regelsatz", true);
+        this.standardFields.put("images", true);
+        this.tifHeaderDocumentName = "";
+        this.tifHeaderImageDescription = "";
+    }
+
+    public Map<String, Boolean> getStandardFields() {
+        return this.standardFields;
+    }
+
+    /**
+     * Get all document types.
+     *
+     * @return list of ConfigOpacDoctype objects
+     */
+    public List<ConfigOpacDoctype> getAllDoctypes() {
+        return ConfigOpac.getAllDoctypes();
```

Comment:
The possible doctypes should be a selection of all possible divisions from the rule set. You get it like this:

    public List<SelectItem> getAllDoctypes() {
        return createProcessForm.getRuleset()
            .getStructuralElements(createProcessForm.getPriorityList()).entrySet()
            .stream().map(entry -> new SelectItem(entry.getKey(), entry.getValue()))
            .collect(Collectors.toList());
    }